### ISSUE TITLE
Acronyms

### ISSUE THREAD

**Issue by IntegraMOD** [phpbbireland/phpbb3-extension-portal#3]
BTW this and the last two issues are from tylers repo and im running it on phpBB3.3.11 and php8.2. 

with the portal enabled, making a post with any word recognized as an acronym results in raw <acronym code being posted even with parse acronyms disabled in the portal acp

**Comment by Prosk8er** [comment]
yeah i have this noted here https://gotskillslounge.com/test/viewtopic.php?t=2
haven't had a chance to dig back into this yet
